### DISCUSSION TITLE
Manage Hedy video content

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#3914] [Community]
A while ago, I made a bit of progress on an online course for teachers:

* I made some videos for an online course, see: https://www.youtube.com/@hedy5423/videos
* I created a weekly storyline with the videos using a google form in Dutch (f.e. this one: https://docs.google.com/forms/d/e/1FAIpQLScWckebdK1TuxWKowXEgWy5QaUzXLQDKNPvSlskzDHHjaO6JQ/viewform)
* I created a MailChimp automation to send the weekly forms to Dutch teachers that subscribed.

Then due to the fact I was busy and my notorious lack of follow through on things 🤣, I did not pursue this any further. But we should! Here are some things that would be great to be picked up by someone:

* Read the feedback we gathered and improve the storyline (Dutch needed)
* Translate the forms to English
* Update the automation to send the mails in English
* Ask teachers at signup if they want to participate (requires some programming but not much) and then mark them in Mailchimp so they get the videos weekly.

Related (maybe we should split this into a separate issue or discussion) we need to place the videos somewhere easily findable, in the teachers manual for example? A textual explanation of features + a video would be great!

(And sadly of course, as happens with videos, since the new website design they are a bit outdated but they are still useful, and when we have figured out how to place them, I can always make a few new ones)

**Comment by Mark-Giesen** [comment]
In my head this would merge nicely with #3916

**Comment by SabinaChita** [comment]
I would like to pick up on this one also #3916

**Reply by Felienne** [reply]
Great!! This too can be split in several issues (let me know if you need help)
